NEWS

```diff
@@ -1,5 +1,5 @@
 #
-## Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2001-2021. ALL RIGHTS RESERVED.
+## Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2001. ALL RIGHTS RESERVED.
```

Comment:
shouldn't we add -2024?

Reply:
just saw previous comments

Reply:
It should be only the year of creation according to the guidance.